application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -235,48 +232,18 @@ private static void handleErrors(@NotNull SlashCommandEvent event,
 
     /**
      * This grabs the OptionMapping, after this it <br />
-     * - validates whenever it's within an {@link Integer Integer's} range <br />
-     * - validates whenever it's positive <br />
+     * - validates whenever It's null or not <br />
      *
      * <p>
      * <p/>
      *
-     * @param event the {@link SlashCommandEvent}
-     * @param optionMapping the {@link OptionMapping}
-     * @return nullable {@link Integer}
-     * @throws java.lang.IllegalArgumentException if the option's value is - outside of
-     *         {@link Integer#MAX_VALUE} - negative
+     * @param optionMapping the {@link OptionMapping} to validate
      */
```

Comment:
The Javadoc is in a bad state right now, the text is hard to understand and not fitting. Also, what is this `<p><p/>` mess? And why do you take in the `event`, it is not even used!

Rewrite:
```java
* Interprets the option as integer, if present.
*
* @param option the option to interpret or null if not present
* @return the integer contained in the option, if present
```